retriever/lib/repository.py

```diff
@@ -55,7 +59,8 @@ def check_for_updates():
             path_script_name = os.path.normpath(os.path.join(HOME_DIR, "scripts", script_name))
```

Comment:
Rename path_script_name to script_home_path
```python
            script_home_path = os.path.normpath(
                os.path.join(HOME_DIR, "scripts", script_name)
            )
```
Move these `os path joins` in this for loop here and use variable names instead, just like above
 in `_download_from_repository("scripts/", + script_name, os.path.normpath(os.path.join(SCRIPT_WRITE_PATH, script_name)),`
`os.remove(os.path.normpath(os.path.join(HOME_DIR, "scripts", script_name)))`

This file should have no in above the length of 90